tests/test_quickstart.py

```diff
@@ -0,0 +1,125 @@
+# imports for example data and model training
+from credoai.datasets import fetch_creditdefault
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.model_selection import train_test_split
+
+from abc import ABC
+
+from credoai.artifacts import ClassificationModel, TabularData
+from credoai.evaluators import (
+    ModelFairness,
+    Performance,
+    Privacy,
+)
+from credoai.lens import Lens
+
+
+def setup_artifacts():
+    data = fetch_creditdefault()
+    df = data["data"]
+    df["target"] = data["target"].astype(int)
+
+    # fit model
+    model = RandomForestClassifier(random_state=42)
+    X = df.drop(columns=["SEX", "target"])
+    y = df["target"]
+    sensitive_features = df["SEX"]
+    (
+        X_train,
+        X_test,
+        y_train,
+        y_test,
+        sensitive_features_train,
+        sensitive_features_test,
+    ) = train_test_split(X, y, sensitive_features, random_state=42)
+    model.fit(X_train, y_train)
+
+    return (
+        X_train,
+        y_train,
+        X_test,
+        y_test,
+        sensitive_features_train,
+        sensitive_features_test,
+        model,
+    )
+
+
+class Base_Evaluator_Test(ABC):
```

Comment:
Is this class necessary, since it is empty?

Reply:
Removed

Re run notebook code: jupyter is not presently in our dependency list for lens. There's maybe a good reason that it shouldn't be (e.g. Lens is just a python package). We'd have to separately add the dependency specifically for the tests.